lib/user.rb

```diff
@@ -0,0 +1,56 @@
+require "./lib/databaseconnection.rb"
+require "bcrypt"
+
+class User
+
+attr_reader :name,:password, :email, :id
+
+  def self.create(name:, password:, email:, receive_email:)
```

Comment:
A few of the methods in this class feel a bit busy and there's some repeated code across them - potentially a candidate for some small refactors?